homeassistant/components/glances/sensor.py

```diff
@@ -352,14 +352,7 @@ def __init__(
     @property
     def available(self) -> bool:
         """Set sensor unavailable when native value is invalid."""
-        if super().available:
-            return (
-                not self._numeric_state_expected
-                or isinstance(value := self.native_value, (int, float))
-                or isinstance(value, str)
-                and value.isnumeric()
-            )
-        return False
+        return super().available and self._attr_available
```

Comment:
This is misusing `_attr_available` IMO, I think we should use a different attribute, maybe `_data_present` or something.

Reply:
Thanks for the feedback, I'll have a look into this.

Reply:
I have created an attribute `_data_valid` to replace the use of `_attr_available`